Doc/whatsnew/3.10.rst

```diff
@@ -1774,6 +1774,16 @@ Build Changes
 C API Changes
 =============
 
+PEP 652: Maintaining the Stable ABI
+-----------------------------------
+
+The Stable ABI (Application Binary Interface) for extension modules or
+embedding Python is now explicitly defined.
+C API and ABI stability guarantees and best practices for using the Stable ABI
+are described in :ref:`stable`.
```

Comment:
Minor nit: when I first read this I was slightly confused and re-read a few times, maybe because of the many "and" s, I suggest wording it like this:
```suggestion
:ref:`stable` describes C API and ABI stability guarantees along with best
practices for using the Stable ABI.
```

Reply:
Thanks, that does read better!